app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable

```

Comment:
- [ ] Kindly ensure that the application asks the user for email confirmation. Please include the Devise `:confirmable` option.
![image](https://github.com/rodrigopenayo98/Blog-App-Rod/assets/46839548/8e2a0354-015a-4376-b90a-1b9f56781ca8)